CHANGELOG.md

```diff
@@ -1,6 +1,37 @@
 # Change Log
 
-## [Unreleased](https://github.com/trailofbits/manticore/compare/0.2.4...HEAD)
+## [Unreleased](https://github.com/trailofbits/manticore/compare/0.2.5...HEAD)
+
+## 0.2.5 - 2019-03-18
+
+Thanks to our external contributors!
+
+ - [werew](https://github.com/trailofbits/manticore/commits?author=werew)
+ - [NicolaiSoeborg](https://github.com/trailofbits/manticore/commits?author=NicolaiSoeborg)
+ - [Joool](https://github.com/trailofbits/manticore/commits?author=Joool)
+
+### Ethereum
+
+* **[added API]** `json_create_contract` - support creating EVM contracts from Truffle JSON artifacts ([#1376](https://github.com/trailofbits/manticore/pull/1376))
+* **[changed API]** Moved default gas value to config module ([#1346](https://github.com/trailofbits/manticore/pull/1346))
+* **[fixed API]** Fixed account creation with a code field ([#1371](https://github.com/trailofbits/manticore/pull/1371))
+* **[fixed API]** Fixed an incorrect attribute in `last_return` ([#1341](https://github.com/trailofbits/manticore/pull/1341))
+* **[removed API]** Inlined get_possible solutions function as it's only meant to be used once internally ([#1372](https://github.com/trailofbits/manticore/pull/1372))
```

Comment:
I'd call it `refactor` or `improvement` as this function was not a public API (note that it is defined inside of another function)
```suggestion
* **[refactor]** Inlined get_possible solutions function as it's only used once ([#1372](https://github.com/trailofbits/manticore/pull/1372))
```